simpleline/render/adv_widgets.py

```diff
@@ -175,3 +176,44 @@ def input(self, args, key):
 
     def prompt(self, args=None):
         return Prompt(_("Press %s to return") % Prompt.ENTER)
+
+
+class GetInputScreen(UIScreen):
+    """Screen for getting user input."""
+
+    def __init__(self, prompt, condition=None):
```

Comment:
Also the callback `condition` could have a more specific name. For example `accept_input` or `input_handler`.

Reply:
It is prompt because it is printed before user input. However, it could be even message to prevent confusion.

I would go for `accept_condition` that sound better to me.

Reply:
Ok.

Reply:
Fixed.